language-guides/ruby/ruby.adoc

```diff
@@ -20,118 +20,147 @@ You should be familiar with link:/developer/get-started/graph-database[graph dat
 
 toc::[]
 
+include::../_includes/sample_project.adoc[]
+
 // tag::intro[]
```

Comment:
if you remove the end-tag then you have to remove this too, or add the end-tag back in.